rules/integrations/kubernetes/execution_pod_created_in_kube_namespace

```diff
@@ -0,0 +1,56 @@
+[metadata]
```

Comment:
Missing extension on this file, needs linting

Reply:
oops, thank you